src/ZODB/ExportImport.py

```diff
@@ -158,18 +158,23 @@ def persistent_load(ooid):
                 oids[ooid] = oid = self._storage.new_oid()
                 return_oid_list.append(oid)
 
-            # Blob support
-            blob_begin = f.read(len(blob_begin_marker))
-            if blob_begin == blob_begin_marker:
+            if (len(data) < 99 and b'blob' in data and
```

Comment:
99 seems a bit magic.  I'd like to see it defined as a named constant at the top level, with a comment explaining why a pickled blob will never be longer than that.

Reply:
99 is arbitrary (it's a Jim's number ;-) ).  It's chosen to be bigger than I'd expect a blob record to ever be. It's a micro optimization to avoid searching for the test 'blob' in a larger pickle.  Maybe we should just drop it.

I hate having to look at the top of a file to see what something means.  (That's why I dislike from imports.)

I'll just remove it.